.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
`node_modules` is better ignored from a more global location. Can you remove this file?

Reply:
Okay